portfolio/src/main/java/com/google/sps/servlets/JsonDataServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.JsonData;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Date;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/json")
+public final class JsonDataServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    String Brand = "Toyota";
+    String Model = "Corolla";
+    String Color = "Red";
+    Integer Year = 2019;
```

Comment:
Based on the [Java style guide](https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names), local variable names should start with a lowercase letter.
```suggestion
    String brand = "Toyota";
    String model = "Corolla";
    String color = "Red";
    int year = 2019;
```